go/libkb/login_state.go

```diff
@@ -1083,6 +1084,25 @@ func (s *LoginState) Keyring(h func(*SKBKeyringFile), name string) error {
 	return err
 }
 
+func (s *LoginState) MutateKeyring(h func(*SKBKeyringFile) *SKBKeyringFile, name string) error {
+	var err error
+	aerr := s.Account(func(a *Account) {
+		var kr *SKBKeyringFile
+		kr, err = a.Keyring()
+		if err != nil {
+			return
+		}
+		if h(kr) != nil {
+			a.ClearKeyring()
```

Comment:
Could you add a comment that this is just clearing the keyring cache?  Or change function name.  On my first read, I thought it was erasing the keyring.